### PR TITLE
Update meeting agenda item status

### PR DESCRIPTION
**Pull request by jerrycyip**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
  * Update functionality to better align with latest [MVP designs](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=6150-89385&mode=design&t=a2t3EN7Iytj0m7dP-0) and [MVP Components](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=5537-109157&mode=design&t=b8Spb17FnUVyw5Ku-0)

- Describe what changes are being made
  * In line with latest [MVP designs](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=6150-89385&mode=design&t=a2t3EN7Iytj0m7dP-0) and [MVP Components](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=5537-109157&mode=design&t=b8Spb17FnUVyw5Ku-0), update Meeting Agenda Item Status selection functionality for admins to implement re-usable dropdown from prior [PR 336](https://github.com/codeforsanjose/project-happening-atm/pull/336), deprecate redundant code and refresh styling to match designs.  In addition, in contrast to prior functionality, the new dropdown is now more user friendly by indicating to the user which option is hovered over for selection (e.g. style changes: option dims and text has shadow etc) as well as which option is currently selected (similar style differentiation).  Similarly, the new dropdown selector doesn't obscure/grey out the entire page when open as was previously implemented (via a modal), in effect preventing the admin from viewing any other meeting details.

- Describe why these changes are being made
  * Updates to styling and functionality align with latest [MVP designs](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=6150-89385&mode=design&t=a2t3EN7Iytj0m7dP-0) and [MVP Components](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=5537-109157&mode=design&t=b8Spb17FnUVyw5Ku-0).  
  * Following best practice, frontend components such as the admin dropdown selector should be re-usable whenever possible (e.g. DRY code = don't repeat yourself).  In addition, in contrast to prior functionality, the new dropdown is now more user friendly by indicating to the user which option is hovered over for selection (e.g. style changes: option dims and text has shadow etc) as well as which option is currently selected (similar style differentiation).  Similarly, the new dropdown selector doesn't obscure/grey out the entire page when open as was previously implemented (via a modal), in effect preventing the admin from viewing any other meeting details.

- List the use cases and edge cases relevant to this PR
  * Updates to styling and functionality also align with latest [MVP designs](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=6150-89385&mode=design&t=a2t3EN7Iytj0m7dP-0) and [MVP Components](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=5537-109157&mode=design&t=b8Spb17FnUVyw5Ku-0)

- Describe how errors will be handled. How will we know if this code breaks in production
  * N/A - functionality tested before/after code changes for any errors

- Describe any external libraries/dependencies added or removed in this PR
  * N/A

- Describe any security risks are there regarding this change
  * N/A

- Describe how you tested these changes
  * Functionality tested before/after code changes for any errors
  * Note, the MVP designed confirmation modal that is shown after updating the meeting status will be included in a later (hopefully next) PR submission. That modal functionality calls for the creation of a new re-usable confirmation modal that can be re-used across different confirmation use cases. Upon reviewing the existing code base, there currently appears to be multiple modal components implemented all doing slight variations of the same thing (not good).

- Link to relevant external documentation
  * Updates to styling and functionality also align with latest [MVP designs](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=6150-89385&mode=design&t=a2t3EN7Iytj0m7dP-0) and [MVP Components](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?type=design&node-id=5537-109157&mode=design&t=b8Spb17FnUVyw5Ku-0)

--------------------------
### :clipboard: Mandatory Checklist

- [x] Example of a checked item (please remove when creating your Pull Request) 

- [x] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [x] Have you discussed these changes with the project leader(s)?
- [x] Do all variable and function names communicate what they do?
- [x] Were all the changes commented and / or documented?
- [x] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [x] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [x] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### Before

Admin user:

https://github.com/codeforsanjose/project-happening-atm/assets/54079796/18d51a1f-8c4b-45d0-bf94-887f319397fc

Community User:
mtg_agenda_item_status_before community_user

#### After

Admin user:

https://github.com/codeforsanjose/project-happening-atm/assets/54079796/129de4e2-d39f-43ac-81cd-49493c691596

Community User:

mtg_agenda_item_status_after community_user

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

**Review by jerrycyip — COMMENTED** [review]
for some reason github is showing backend/graphql_api/graphql/resolvers/mutation.js as a new file, but it's the same file as before.  I tried adding a comment and re-pushing the file, but it appears to show the same git tracking issue (6575da9 above).  I think the real issue is the ci / build errors for AWS below?

File: frontend/src/components/MeetingView/AgendaView/AgendaView.jsx
Comment (jerrycyip) [inline frontend/src/components/MeetingView/AgendaView/AgendaView.jsx:1]: showCompleted no longer in scope as per latest MVP designs - hence removed

File: frontend/src/components/MeetingListView/MeetingListView.jsx
Comment (jerrycyip) [inline frontend/src/components/MeetingListView/MeetingListView.jsx:1]: apologies, I think the majority of the 'differences' for MeetingListView.jsx are actually just the linter applying auto-formatting upon save.

File: backend/graphql_api/graphql/resolvers/mutation.js [hunk 1]
```diff
@@ -379,7 +379,7 @@ module.exports = (logger) => {
     //Convert email to lowerCase
     email_address = email_address.toLowerCase().trim();
     if (context.token === null) {
-      const isAdmin = await authentication.verifyAdmin(dbClient, email_address);
+      const isAdmin = await authentication.verifyAdmin(dbClient, email_address); // local dev hack: change to true for admin user creation
```
Comment (JMStudiosJoe) [inline backend/graphql_api/graphql/resolvers/mutation.js:382]: awesome

File: frontend/src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -39,8 +36,16 @@ function Footer() {
           </button>
         </div>
         <div className="logos">
-          <a href="https://www.onlyinsj.org/" target='_blank' rel="noreferrer"><img className="OSJ" src={OSJ_Logo} alt='Only in San Jose'/></a>
-          <a href="https://www.codeforsanjose.org/" target='_blank' rel="noreferrer"><img className="CFSJ" src={CFSJ_Logo} alt='Code for San Jose'/></a>
+          <a href="https://www.onlyinsj.org/" target="_blank" rel="noreferrer">
+            <img className="OSJ" src={OSJ_Logo} alt="Only in San Jose" />
+          </a>
+          <a
+            href="https://www.codeforsanjose.org/"
```
Comment (JMStudiosJoe) [inline frontend/src/components/Footer/Footer.jsx:43]: this needs to be www.opensourcesanjose.org now

File: frontend/src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -39,8 +36,16 @@ function Footer() {
...
+            target="_blank"
+            rel="noreferrer"
+          >
+            <img className="CFSJ" src={CFSJ_Logo} alt="Code for San Jose" />
```
Comment (JMStudiosJoe) [inline frontend/src/components/Footer/Footer.jsx:47]: but can keep the nameing here hhe he he

File: frontend/src/components/Footer/Footer.scss [hunk 4]
```diff
@@ -14,8 +14,7 @@
 }
 
 footer {
-  
-  z-index: 9000;
+  z-index: 5000; // allow admin agenda item dropdown to sit on top
```
Comment (JMStudiosJoe) [inline frontend/src/components/Footer/Footer.scss:17]: i never like the huge number so far i think this is 1 of like 4 so far so only need to keep it max under 100

**Review by JMStudiosJoe — CHANGES_REQUESTED** [review]
small change rquests and great job!

File: frontend/src/components/Footer/Footer.scss (hunk 4 above)
Comment (jerrycyip) [reply frontend/src/components/Footer/Footer.scss:17]: yeah, the large numbers see excessive to me as well.  I can definitely lower it and resubmit.  Thanks for the review and feedback!

File: frontend/src/components/Footer/Footer.jsx (hunk 2 above)
Comment (jerrycyip) [reply frontend/src/components/Footer/Footer.jsx:43]: Noted - will update.  Thanks!

File: frontend/src/components/Footer/Footer.jsx (hunk 3 above)
Comment (jerrycyip) [reply frontend/src/components/Footer/Footer.jsx:47]: Sounds good. Once we have new logo images handy, I can update this in a future PR.

File: frontend/src/components/Footer/Footer.scss (hunk 4 above)
Comment (jerrycyip) [reply frontend/src/components/Footer/Footer.scss]: Per suggestions, I have reviewed all z-indexes above >10, and have updated values as follows:
Index.js: (instance of z-index w/ old value of) 10000 -> 60 (new value)
Dropdown.scss: 10000 -> 50
NavbarHeader.scss:9999 -> 50
footer.scss: 5000 -> 40
Subscribe.scss:  3000 ->  30
Search.scss:  1001 -> 20
meetingHeader.scss: 100 -> 10

File: frontend/src/components/Footer/Footer.jsx (hunk 3 above)
Comment (JMStudiosJoe) [reply frontend/src/components/Footer/Footer.jsx:47]: lets have it as another pr to just update the logo, no worries to have here

File: frontend/src/index.scss [hunk 5]
```diff
@@ -66,7 +66,7 @@ body {
   bottom: 0;
   display: grid;
   place-items: center;
-  z-index: 10000;
+  z-index: 60;
```
Comment (JMStudiosJoe) [inline frontend/src/index.scss:69]: omg yes so much easier hahahahahah OVER 9000?!?!

**Review by JMStudiosJoe — APPROVED** [review]

**Pull request merged**